best-practices/README.md

```diff
@@ -161,6 +161,8 @@ Postgres
 * Consider a [partial index] for queries on booleans.
 * Constrain most columns as [`NOT NULL`].
 * [Index foreign keys].
+* Use an `ORDER BY` clause on queries where the results will be displayed to a
+  user, as queries without one may return results in changing, arbitrary order.
```

Comment:
I meant before "changing". I could take or leave "those results" -- I think just "results" is more clear.